.github/workflows/build_deploy.yml

```diff
@@ -51,11 +51,12 @@ jobs:
         env:
           # Python 2.7 not supported
           # Python 3.5 Deprecated
+          # Python 3.6 Deprecated
```

Comment:
Perhaps modify here as well:
https://github.com/planetmarshall/pillow-jpls/blob/4c30bae73836d02d2500cb31e832ca18cfdcdf76/setup.py#L53

Reply:
This message is just a warning, not a failure:
```
cibuildwheel 2.x no longer supports Python < 3.6. Please use the 1.x series or update CIBW_SKIP
```
Python < 3.6 is not supported and will be skipped any way, so you can remove obsolete versions **`cp27-* cp35-*`** from the CIBW_SKIP skip list.